### PR TITLE
add Min_issue query to metadata and add 3 tests

### PR DESCRIPTION
**Pull request by zhuoran-Cheng16**
### Summary

- Add min_issue query by using JOIN to combine with original inner_sql, modified 3 test file by adding some new test values without the is_latest_issue to test if it can get the correct min_issue. Already pass all the unit test and integration test.

### CODE REVIEW

File: src/acquisition/covidcast/database.py [hunk 1]
```diff
@@ -270,36 +270,60 @@ def compute_covidcast_meta(self, table_name='covidcast', use_index=True):
...
-        MIN(`value`) AS `min_value`,
-        MAX(`value`) AS `max_value`,
-        ROUND(AVG(`value`),7) AS `mean_value`,
-        ROUND(STD(`value`),7) AS `stdev_value`,
-        MAX(`value_updated_timestamp`) AS `last_update`,
-        MAX(`issue`) as `max_issue`,
-        MIN(`lag`) as `min_lag`,
-        MAX(`lag`) as `max_lag`
+      SELECT 
+        inner_main_sql.*,inner_min_sql.*
```
Comment (krivard) [inline src/acquisition/covidcast/database.py:274]: do you need all the columns from inner_min_sql to be returned? if not, you can do
```suggestion
        inner_main_sql.*,inner_min_sql.min_issue     
```
-- a column doesn't have to be in the outer SELECT to be used in JOIN .. ON

File: src/acquisition/covidcast/database.py [hunk 2]
```diff
@@ -312,7 +336,7 @@ def worker():
       try:
         while True:
           (source, signal) = srcsigs.get_nowait() # this will throw the Empty caught below
-          w_cursor.execute(inner_sql, (source, signal))
+          w_cursor.execute(inner_sql, (source, signal,source, signal))
```
Comment (krivard) [inline src/acquisition/covidcast/database.py:339]:
```suggestion
          w_cursor.execute(inner_sql, (source, signal, source, signal))
```

**Review by krivard — COMMENTED** [review]
Final approval pending scalability results, but in the meantime we can probably cut the number of columns returned in half

File: src/acquisition/covidcast/database.py (hunk 1 above)
Comment (zhuoran-Cheng16) [reply src/acquisition/covidcast/database.py:274]: I will fix that problem soon! Thanks for pointing out.

**Comment by krivard** [comment]
Summary from offline discussion -- 

this is not as performant as we'd like on dense datasets like doctor-visits and jhu-csse, and splitting the JOIN into two queries doesn't seem to help. instead, we'd like to pursue an approach based on the incremental metadata PR (#375), in which we would figure out the min issue once for everything, then update it at _acquisition time_ to account for new earlier issues (as batch uploads) or new source+signal pairs (as part of initial metadata estimates).

we'll return to this PR to either modify or close it once we've done some more work on the incremental meta code.

**Comment by krivard** [comment]
wontfix; invalidated by v4

**Pull request closed without merging**